spec/components/bali/link_spec.rb

```diff
@@ -63,11 +63,26 @@
     expect(rendered_component).to have_css 'a.is-active', text: 'Click me!'
   end
 
-  it 'renders a link with turbo method' do
-    @options.merge!(method: :post)
+  context 'with the method parameter' do
+    context 'when the method is not get' do
+      it 'renders a link with turbo method' do
+        @options.merge!(method: :post)
 
-    render_inline(component)
+        render_inline(component)
+
+        expect(rendered_component).to have_css 'a[data-turbo-method="post"]', text: 'Click me!'
+      end
+    end
 
-    expect(rendered_component).to have_css 'a[data-turbo-method="post"]', text: 'Click me!'
+    context 'when the method is get' do
+      it 'renders a link without turbo method' do
+        @options.merge!(method: :get)
+
+        render_inline(component)
+
+        expect(rendered_component).to have_css 'a.link-component', text: 'Click me!'
+        expect(rendered_component).not_to have_css 'a[data-turbo-method="post"]'
```

Comment:
te falta validar que se agregue el data-method="get"

```suggestion
        expect(rendered_component).to have_css 'a.link-component', text: 'Click me!'
        expect(rendered_component).to have_css 'a[data-method="get"]'
        expect(rendered_component).not_to have_css 'a[data-turbo-method="post"]'
```